pvr.vuplus/addon.xml.in

```diff
@@ -175,6 +175,10 @@
     <disclaimer lang="zh_TW">這是測試版軟體！其原創作者並無法對於以下情況負責，包含：錄影失敗，不正確的定時設定，多餘時數，或任何產生的其它不良影響...</disclaimer>
     <platform>@PLATFORM@</platform>
     <news>
+v3.12.5
+- Fixed: Large refactor for code organisation
```

Comment:
Is refactoring a fix? I hope it does not change functionality. ;-)

Reply:
I’ve just been using fixed and added. I guess neither are appropriate.

Reply:
I could argue that good refactoring could be deemed as a fix ;) ...